### DISCUSSION TITLE
Migrazione da HA Docker a HA Operating System

### DISCUSSION THREAD

**Discussion by papinist** [indomus/forum#498] [HUB: Home Assistant]
Ciao a tutti
vorrei passare il mio Home Assistant Docker su una installazione pulita di Home Assistant Operating System.
E' sufficiente creare uno snapshot, installare la nuova versione e lì ripristinare lo snapshot?
Al momento sono alla 2021.6.6 e ho visto che è uscita la 2021.7.1 , però non posso aggiornare per i problemi con la vecchia versione di docker. 
E' possibile ripristinare lo snapshot della 2021.6.6 sulla 2021.7.1?

**Comment by snakuzzo** [comment]
Se hai HA Core su Docker non esistono gli snapshot (e ti sconsiglio vivamente di passare su HassOS)
Se invece hai installato HA Supervised su Docker allora sì...dovrebbe funzionare.

Ad ogni modo puoi risolvere semplicemente aggiornando Docker e installando la libreria `libseccomp` (come indicato nelle release note)

![image](https://user-images.githubusercontent.com/7205371/124997925-c956a580-e04b-11eb-8edf-fd7b8b0ea397.png)

**Comment by papinist** [comment]
Sì ovviamente ho gli snapshot.. non so come si chiama la versione, immagino supervised come hai detto, ma ricordo che quando l'ho installata per la prima volta c'era questa cosa di usare la versione 'su docker' perchè avrebbe permesso di usare il raspberry anche per altro.

Visto che però con il tempo ho capito che lo lascerò dedicato a HA, vorrei mettere appunto una versione 'più lineare' per chi non mastica il mondo unix come me, e che è anche quella raccomandata dal team di HA.

Avevo visto che in teoria è sufficiente aggiornare docker e quella libreria, ma non avendo idea di come si faccia preferisco eliminare del tutto docker che non mi porta nessun vantaggio.

**Reply by snakuzzo** [reply]
Ok...allora fai lo snapshot e poi lo riporti su HassOS
P.S. Non avviare un nuovo thread quando rispondi...se devi rispondere fallo sotto il messaggio a cui vuoi rispondere

**Reply by papinist** [reply]
nuovo thread? Non mi sembrava.. sono sempre all'interno della discussione che ho aperto. le due caselle di risposta non hanno nessuna indicazione specifica..

Ok domani provo a fare il tutto, grazie!

**Reply by snakuzzo** [reply]
Sì ma vedi bene...che nella stessa discussione puoi avviare più thread.
Per intenderci...avresti dovuto rispondere qui...

![image](https://user-images.githubusercontent.com/7205371/125000440-b09cbe80-e050-11eb-89ca-b854c03f327c.png)

**Reply by papinist** [reply]
Ho visto, e ho capito.. ma quale è l'utilità? Ci si raccomanda sempre di non andare OT e poi si aprono più discussioni nello stesso thread :) Certo sempre meglio questo forum della "chat" precedente eh :-D almeno ci sono i tasti per inserire codice, grassetto ecc senza doversi ricordare a mente come fare.
Non succederà mica come sul forum di teslari che capita che qualcuno risponda ad un messaggio che non è l'ultimo, e ti trovi le risposte sparse per tutto il thread invece che ordinate in fondo..

Opinioni personali ovviamente!

**Reply by snakuzzo** [reply]
Discussions è fatto così... fine :)

**Reply by papinist** [reply]
Volevo concludere dicendo che è andato tutto bene, anche se la versione di HA OS 6.1 una volta installata, aveva dentro la 2021.6.6 identica a quella dalla quale arrivavo, quindi non posso dire con certezza che il backup avrebbe funzionato su una più recente.
Per chi si cimenterà in questo passaggio, faccio notare che al primo avvio di HA OS _non_ dobbiamo usare https altrimenti non si collega, mentre al riavvio dopo aver ripristinato il backup, deve essere usato https altrimenti non si collega.
